client/js/widgets/widget.js

```diff
@@ -2302,6 +2312,18 @@ export class Widget extends StateManaged {
       await this.set('rotation', degrees);
   }
 
+  async selected() {
```

Comment:
~~These do not need to be `async`. Basically only if they contain an `await`.~~

Scratch that. See below.